Configurations/00-base-templates.conf

```diff
@@ -28,7 +28,7 @@ my %targets=(
 	build_file	=> "Makefile",
 
 	AR		=> "ar",
-	ARFLAGS		=> "r",
+	ARFLAGS		=> "cru",
```

Comment:
Why aren't we using the `q` option?  It appends without checking for duplicates but avoids quadratic time behaviour.  THe build system should be able to deal with ensuring that the contents unique.

Reply:
Yes, please change this to "qc".
(either way, this is wrong, as the command letter is "r", while "c" and "u" are modifiers, so this should have been "r u")

Reply:
... never mind the parenthesis, order isn't as important as I thought...  however, "qc" would still be better

Reply:
All right, changed to the more efficient "qc".

Reply:
Anyway, it looks like the stability problem (due to duplicate entries in the archive) cannot be solved anyway at the level of `ar` options.